src/components/CityColumnChart/CityColumnChart.js

```diff
@@ -46,22 +46,25 @@ function sortChartData(cities, indicator) {
   return sortedCities;
 }
 
-const CityBarChart = (props) => {
-  // If more than one indicator is passed in, this becomes a stacked bar chart
+const CityColumnChart = (props) => {
+  // If more than one indicator is passed in, this becomes a stacked column chart
   const isStacked = props.indicatorIds.length > 1;
 
+  // To ensure readability for various config options
+  const narrowViewport = document.documentElement.clientWidth < 650;
```

Comment:
Minor: `window.innerWidth` is shorter, more traditional, but generally getting the screen width to define the display in JavaScript is a bad idea, more comments below.